fs/filesize.py

```diff
@@ -34,7 +34,8 @@ def _to_str(size, suffixes, base):
     elif size < base:
         return "{:,} bytes".format(size)
 
-    for i, suffix in enumerate(suffixes, 2):
+    # TODO (dargueta): Don't rely on unit or suffix being defined in the loop.
```

Comment:
As long as there is at least one value in `suffixes` we can rely on those variables being defined. Perhaps an `assert len(suffixes)` is appropriate?

Reply:
That'd work.

Reply:
So actually I found an edge case -- this will break for values above 10<sup>24</sup> bytes (which is absurdly high) because it'll run off the edge of the array and `unit` will never be defined.

While incredibly unlikely to be encountered in practice, a troll could still use this to break things. I've modified it to both get rid of the linter warning *and* behave correctly in the event this gets handed a huge number.

Reply:
I don't see how it will break. It will use the last unit, which may result in an absurdly large value, bit it's still valid.

```python
>>> from fs.filesize import traditional
>>> traditional(1E32)
'82,718,061.3 YB'
>>> traditional(1E64)
'8,271,806,125,530,276,925,072,459,481,976,663,965,696.0 YB'
```

Reply:
Odd...broke for me. o.O

Reply:
I removed the change btw